django_tasks/backends/immediate.py

```diff
@@ -37,14 +37,21 @@ def enqueue(
         try:
             result = json_normalize(calling_task_func(*args, **kwargs))
             status = ResultStatus.COMPLETE
-        except Exception as e:
+        except BaseException as e:
             try:
                 result = exception_to_dict(e)
             except Exception:
                 logger.exception("Task id=%s unable to save exception", result_id)
                 result = None
+
+            # Use `.exception` to integrate with error monitoring tools (eg Sentry)
+            logger.exception("Task execution failed: %s", e)
```

Comment:
Not sure if this message could be improved or not. I was looking at [these lines from db_worker](https://github.com/RealOrangeOne/django-tasks/blob/master/django_tasks/backends/database/management/commands/db_worker.py#L126-L132) as reference:

```python
# Use `.exception` to integrate with error monitoring tools (eg Sentry)
logger.exception(
    "Task id=%s path=%s state=%s",
    db_task_result.id,
    db_task_result.task_path,
    ResultStatus.FAILED,
)
```

Reply:
`task.module_path` should do it. I agree it'd be nice to use `logfmt` formatting if we can.

Reply:
There's a module path on the `Task` you can use.

`logfmt` would also be great - there should be everything available here to replicate the message, you just might need to move the `TaskResult` creation above the execution for convenience.

Reply:
Pushed an improvement, I just used the `result_id` for the ID, is that fine?